config/peerpods/peerpodssecret.yaml

```diff
@@ -9,3 +9,4 @@ stringData:
   #LIBVIRT_URI: "qemu+ssh://root@192.168.122.1/system?no_verify=1"
   #LIBVIRT_NET: "default
   #LIBVIRT_POOL: "default"
+  #REDHAT_OFFLINE_TOKEN: ""
```

Comment:
What is this token for? Please add a comment describe the use of this token.

Reply:
This is to download base rhel image, I will add the comment